src/main/webapp/upcoming-events.jsp

```diff
@@ -75,17 +75,11 @@
     pageContext.setAttribute("upcomingEvents", upcomingEvents);
 %>
 
-<%@ include file="upcoming-events-css.jsp" %>
-
-<c:if test="${not empty configuration.customCss}">
-<style>
-    <c:out escapeXml="false" value="${configuration.customCss}" />
-</style>
-</c:if>
-
 <fmt:setLocale value="${ctx.locale}" />
 <fmt:setBundle basename="BlackboardMessagesBundle" />
 
+<!-- Start the page with a Scoped element -->
+<!-- https://github.com/eveoh/blackboard-mytimetable/issues/8 -->
```

Comment:
Nah, care if it is removed by IE. As long as the source file contains the comment we're okay.